pkgs/modules/python/default.nix

```diff
@@ -162,7 +162,7 @@ in
     POETRY_USE_USER_SITE = "1";
     PIP_CONFIG_FILE = pip.config.outPath;
     PYTHONUSERBASE = userbase;
-    PYTHONPATH = "${sitecustomize}:${pip.pip}/${python.sitePackages}";
+    PYTHONPATH = "${sitecustomize}:${pip.sitePackages}";
```

Comment:
Why not

```suggestion
    PYTHONPATH = "${sitecustomize}:${python.sitePackages}";
```

?

Reply:
Pip's lib code is separated into a separate directory in the nix setup.

Reply:
right, but do we not need the `python.sitePackages` to have access to (eg) setuptools?

Reply:
oh wow I guess we really do not have entries for that on `PYTHONPATH` today:

```
~/workspace$ tr : '\n' <<< $PYTHONPATH 
/nix/store/wjh5dp8cqvlaxs7yxmy9amn19d54r9nb-sitecustomize/lib/python/site-packages
/nix/store/dfg63lldbvcj207iyr9z0xw21d8ax02n-python3.11-pip-24.0/lib/python3.11/site-packages
```

Reply:
I see the confusion. `python.sitePackages` actually only yields the string: `lib/python3.11/site-packages`